src/ui/components/Header.css

```diff
@@ -9,10 +9,13 @@
 }
 
 #header .logo {
-  height: 14px;
+  height: 18px;
```

Comment:
The logo felt a bit small at 18x18. I tried 24x24 with a border of 4px to get the logo to be 32x32. I was using the github logo as a reference.  Let me know what you think. Otherwise, code lgtms.